bambi/models.py

```diff
@@ -655,16 +725,20 @@ class Term(object):
         constant (bool): indicates whether the term levels collectively
             act as a constant, in which case the term is treated as an
             intercept for prior distribution purposes.
+        default_priors: PriorFactory object
+        auto_scale (bool): If True (default), use default prior scaled to data.
+            Otherwise use flat prior.
     '''
     random = False
 
-    def __init__(self, model, name, data, categorical=False, prior=None,
-                 constant=None):
+    def __init__(self, name, data, categorical=False, prior=None, constant=None,
+        default_priors=PriorFactory(None), auto_scale=True):
```

Comment:
Values initialized in the signature will point to the same object in memory, and while this may not matter in practice here, it can lead to strange bugs in many cases, so is generally discouraged. Better to set to `None` and then initialize inside the method.

Also, do we need to pass an entire `PriorFactory` object? Can't we move the logic that checks if `auto_scale` is set (and appends `_flat` if so) to `Model._add` or `Model.add`, and then we can just use the `prior` argument instead (if it isn't explicitly set)?